### DISCUSSION TITLE
1.5.0.5

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#494] [Обновления]
## Список изменений
- Исправлен перевод для Coursera
- Обновлен Yandex Protobuf

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.5.0.4...1.5.0.5

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.0.5'>1.5.0.5</a>.</em>

**Comment by VadimTrancelover** [comment]
Пытался найти тему на счёт работы с Yandex диском, не получилось. Пытаюсь запустить перевод на яндекс диске - faild to request video translation. Выполнил условия, написанные в Wiki для данного сайта - изменил настройку для "Добавить TM в политику безопасности CSP". В ютубе все работает. Браузер Google Chrome Версия 122.0.6261.95
![Uploading 2024-03-03_13-38-31.png…]()

**Reply by NullVerdict** [reply]
Сохранить нажимал? И да, картинка не загрузилась.
![309414886-1ed9ad79-4ef5-46f2-84ed-150513420814](https://github.com/ilyhalight/voice-over-translation/assets/24498484/8e9af6dc-e638-4f8f-88d7-ef7dc3ad8880)

**Reply by VadimTrancelover** [reply]
аааа, я делал может быть по старому гайду. Там в этом параметре указано поставить ДА. В картинке ничего важного, там плашка переводчика во время проигрывания видео на яндекс диске с текстом, который я написал в сообщении.
Сейчас я сделал как Вы указали в этом скрине, и всё заработало! Огромное спасибо, Вы сэкономили мне кучу времени) Теперь не придётся все видеоуроки лить на ютуб в закртый доступ)
